### PR TITLE
Add spec and parser for 'wc_-l_.proc.1.mountinfo'

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

Add new spec and parser for file `/proc/1/mountinfo`.

### CODE REVIEW

**Comment by JoySnow** [comment]
Hi @xiangce , could you help review this parser?

**Comment by JoySnow** [comment]
This new spec was changed to command("/usr/bin/wc -l /proc/1/mountinfo") .      
Please help review the updated MR.

**Review by xiangce — APPROVED** [review]
Approved pending the approval of Spec

File: insights/parsers/wc_proc_1_mountinfo.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        37
+
+    Raises:
+        insights.parsers.ParseException: if the command output is empty or
+        unparsable.
+    """
+
+    def parse_content(self, content):
+        if len(content) == 0 or 'No such file or directory' in content[0]:
+            raise ParseException("Error: ", content[0] if content else 'empty file')
```
[the remark this answers is not in the document]
Comment (JoySnow) [reply insights/parsers/wc_proc_1_mountinfo.py]: Thanks @bfahr ,  `SkipComponent` will be better here.

File: insights/parsers/wc_proc_1_mountinfo.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    Examples:
+        >>> type(wc_info)
+        <class 'insights.parsers.wc_proc_1_mountinfo.WcProc1Mountinfo'>
+        >>> wc_info.line_count
+        37
+
+    Raises:
+        insights.parsers.SkipComponent: if the command output is empty or
+        missing file.
```
Comment (xiangce) [inline insights/parsers/wc_proc_1_mountinfo.py:35]: you should add one more tab indent in this line, to make sure the doc is built as expected.

**Pull request merged**